### PR TITLE
Add more stuff to the readme

### PR DESCRIPTION
**Pull request by Red1C3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Now, you have built the sample and run it, without writing any C++ code or openi
 - **We won't use [vcpkg](https://vcpkg.io/)** during this course.
 
 ## Awesome OpenGL Games:
-- [AntWar](https://github.com/YamanQD/AntWare/): Damascus university project 2021-2022
+- [AntWare](https://github.com/YamanQD/AntWare/): Damascus university project 2021-2022
```
Comment (MuhammadSulaiman001) [inline README.md:70]: Isn't it a typo and you regret it?😅

**Comment by MuhammadSulaiman001** [comment]
Thanks for contributing <3

File: README.md (hunk 1 above)
Comment (Red1C3) [reply README.md:70]: Oh no, it's like software, hardware... Antware

Comment (MuhammadSulaiman001) [reply README.md:70]: Hmmmm, love it.. BTW, [this](https://www.reddit.com/r/opengl/comments/rev6gr/just_a_little_rant_about_my_college_still/) was very entertaining🥶.

Comment (Red1C3) [reply README.md:70]: > Hmmmm, love it.. BTW, [this](https://www.reddit.com/r/opengl/comments/rev6gr/just_a_little_rant_about_my_college_still/) was very entertaining🥶.

Indeed 🥶

**Pull request merged**